src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -22,7 +22,7 @@ public void testFindProductsNoResults() {
     }
 
     @Test
-    public void testSaveNewProduct() {
+    public void testSaveNewProduct() throws ProductNotFoundException {
         Currency usd = Currency.getInstance("USD");
```

Comment:
[Optional] you can make currency global constant if you're using it multiple times